openclaw-skill/SKILL.md

```
@@ -0,0 +1,119 @@
+  -d '{
+    "jsonrpc": "2.0",
+    "method": "tools/call",
+    "params": {"name": "TOOL_NAME", "arguments": {ARGS}},
+    "id": 1
+  }'
+```
+
+## 도구 목록
+
+### 시세/검색
+| 도구 | 설명 | 주요 파라미터 |
```

markdownlint-cli2 (0.20.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)